release/e2e.py

```diff
@@ -1637,24 +1659,34 @@ def _run(logger):
                 _process_finished_command(
                     session_controller=session_controller, scd_id=scd_id)
             else:
-                timeout_type = ""
+                error_type = ""
                 runtime = None
                 if isinstance(e, CommandTimeoutError):
-                    timeout_type = "timeout"
+                    error_type = "timeout"
                     runtime = 0
-                elif (isinstance(e, PrepareCommandTimeoutError)
-                      or isinstance(e, FileSyncTimeoutError)
+                    exit_code = 3
+                elif isinstance(e, PrepareCommandTimeoutError):
+                    error_type = "prepare_timeout"
```

Comment:
This will break the existing nightly dashboard. Can we avoid breaking backward compatibility here?